doc/user/content/integrations/connection-pooling.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Connection Pooling"
+description: "How to use connection pooling with Materialize"
+menu:
+  main:
+    parent: "integrations"
+    weight: 10
+    name: "Connection Pooling"
+---
+
+Because Materialize is wire-compatible with PostgreSQL, you can use any
+PostgreSQL connection pooler with Materialize. In this guide, we’ll cover how to
+use connection pooling with Materialize, alongside the common tool PgBouncer.
+
+## PgBouncer
+[PgBouncer](https://www.pgbouncer.org/) is a popular connection pooler for
+PostgreSQL. It provides a lightweight and efficient way to manage database
+connections, reducing the overhead of establishing new connections and
+improving performance.
+
+### Step 1: Install PgBouncer
+
+You can [install PgBouncer](https://www.pgbouncer.org/downloads/) on your local machine or on a server.
+
+### Step 2: Create an authentication userlist file
+
+The userlist file contains the credentials for your Materialize user/app. The file has the format of:
+
+```
+"user@example.com" "mypassword-or-scram-secret"
+```
+
+#### If using `auth_type = plain` (Cloud and Self-Managed)
+
+Specify the password in plaintext:
+
+- **For Cloud**, you can find the password by generating credentials via the [Service Account](https://materialize.com/docs/security/cloud/users-service-accounts/create-service-accounts/) flow.
```

Comment:
minor nit ... since people may already have created credentials ... ,maybe:

**For Cloud**, use the password from an existing [Service Account](/security/cloud/users-service-accounts/create-service-accounts/) or generate a [new one]((/security/cloud/users-service-accounts/create-service-accounts/).